server/core/src/main/scala/sttp/tapir/server/interceptor/decodefailure/DecodeFailureHandler.scala

```diff
@@ -40,6 +40,9 @@ case class DefaultDecodeFailureHandler(
       case None => None
     }
   }
+
+  def withErrorMessage(errorMessageOutput: String => ValuedEndpointOutput[_]): DefaultDecodeFailureHandler =
```

Comment:
since we are modifying the `response` field, maybe this method should also be called `response(messageOutput: String => VEO[_])`?